### ISSUE TITLE
New Design Needed !

### ISSUE THREAD

**Issue by RocktimSaikia** [RocktimSaikia/Contribute-101#37]
Currently the card design of the website is not very clean and synchronous. So need a new design for the card . Here is a rough prototype i was designing for the cards.

![card design](https://user-images.githubusercontent.com/33410545/66466713-038ab180-ea38-11e9-9c47-944368579a22.png)

.This design is just a rough prototype i did while back to get started with the design flow of the website.Will definitely improve it in future.But for now this seems ok.
So anyone wanna work on this is  more than welcome :)

Note : 
1) We are using bootstrap for the style environment but **Do Not** use boostrap's predefined cards for this as bootststaps card is not suitable for dynamic contents.
2)Just use the current `li` tags for the card design .
3) As you see i have added another section in the card design saying **'Fullstack developer'** . So you also have to add a new key value pair as `"role"` to the current object model for the card info  So the object model will look like this : 
```
    name: "",
    country: "",
    tech_stack: ["", "", ""],
    github_username: "",
    twitter_username: "",
    role: ""
```

**Comment by nidsnitesh** [comment]
can i work on this ?

**Comment by jhdcruz** [comment]
> 
> 
> can i work on this ?

I actually have a PR #38 about this, but maybe you can do much better than mine so feel free 😊

**Comment by RocktimSaikia** [comment]
@nidsnitesh Hi there :) . Like @jhdcruz Said you can work on it even if i merge @jhdcruz 's request.Cause this is not a bug or issue, Its enhancement and design so there is always room for new design enhancement.So fee free to work on your design and make a pull request when you are done. Happy coding :)

**Comment by RocktimSaikia** [comment]
I will keep this issue open for new design enhancements in future :)

**Comment by RocktimSaikia** [comment]
1st design enhancement merged #38

**Comment by RocktimSaikia** [comment]
New issue #44 .
